pkg/db/mongodb/plugindb.go

```diff
@@ -174,14 +177,18 @@ func (m *MongoDB) List(ctx context.Context, page db.Pagination) ([]db.Plugin, st
 }
 
 // GetByName gets the plugin with the given name.
-func (m *MongoDB) GetByName(ctx context.Context, name string, filterDisabled bool) (db.Plugin, error) {
+func (m *MongoDB) GetByName(ctx context.Context, name string, filterDisabled, hidden bool) (db.Plugin, error) {
 	ctx, span := m.tracer.Start(ctx, "db_get_by_name")
 	defer span.End()
 
 	criteria := bson.D{
 		{Key: "name", Value: name},
 	}
 
+	if !hidden {
```

Comment:
I think it would be better to name this parameter `filterHidden` cause when `hidden` is true in my mind I would expect to only have an hidden plugin.